__init__.py

```diff
@@ -50,7 +47,7 @@ def __init__(self):
         self.skill_info_api = None
 
         # A variable to turn on/off the example text
-        self.examples_enabled = True
```

Comment:
This should default to `True` so examples are displayed when `skill_info_api` is enabled

Reply:
can we discuss this one? I'd actually like this to be off by default, do people actually like this?

I'm ok with it being toogleable, but most of the time looking at homescreen just sounds damn weird and utterances often show they were not written by a native speaker, specially with the prefix...

whenever i had visits looking at a mk2 this is a comment everyone made

on a side note, this also can be considered advertising according to mycroft :upside_down_face:  
https://www.youtube.com/watch?v=VA0DgYcWwvU

Reply:
This can move to settings; I think right now it defaults to True and becomes false if there isn't a configured skill and OSM import fails

Reply:
id like it to default to False, if someone set up the skill api then they can set the flag to enable this

OSM should always be available too, so the import will usually not fail

Reply:
I have troubles with the config setting `examples_enabled` to get rid of the whole prompt. Something with the variabe not being set in the qml. There is an [issue here](https://github.com/OpenVoiceOS/skill-ovos-homescreen/issues/26) that was closed, but still is valid